File: ICPC/WarmUp 10/ncpc2015solutions/conundrum_Lukas.java
import java.util.*;
import java.util.stream.*;

public class Lukas
{
    public static void main(String[] args)
    {
        final String s = (new Scanner(System.in)).next();
        long res = IntStream.range(0, s.length())
            .filter(i -> "PER".charAt(i % 3) != s.charAt(i)).count();
        System.out.println(res);
    }
}

File: ICPC/WarmUp 10/ncpc2015solutions/conundrum_lukas.go
package main

import "fmt"

func btoi(b bool) int {
    if b { return 1 }
    return 0
}

func main() {
    var s string
    fmt.Scanf("%s", &s)

    res := 0
    for i := 0; i < len(s); i++ {
        if s[i] != "PER"[i % 3] {
            res++
        }
    }
    fmt.Println(res)
}
